README.md

```diff
@@ -120,38 +120,38 @@ like `constant`. The `CharParser` module is a is parser dedicated to char stream
 parsing using another parser.
 
 ```ocaml
-module Utils = Transept_utils.Utils
-module CharParser = Transept_extension.Parser.For_char_list
-module Stream = Transept_stream.Via_parser (CharParser)
-module Genlex = Transept_genlex.Genlex.Make (CharParser)
+module Utils = Transept.Utils.Utils
+module CharParser = Transept.Extension.Parser.For_char_list
+module Stream = Transept.Stream.Via_parser (CharParser)
+module Genlex = Transept.Genlex.Genlex.Make (CharParser)
```

Comment:
Same here for `Transept.Genlex.Genlex`.

Reply:
Genlex has been been renamed as Lexer

Reply:
In which commit? In the text I keep seeing "Genlex.Genlex"